.prettierignore

```diff
@@ -0,0 +1,3 @@
+dist/
+coverage/
```

Comment:
Does this library build into the `dist/` folder and does it produce the `coverage/`? 🤔

Reply:
Yes, it builds into dist/ and I added coverage/ to every repo because that's what I thought Andrei was doing. I don't think it's a problem leaving it there.